### DISCUSSION TITLE
about commit

### DISCUSSION THREAD

**Discussion by eryarsoyyunusemre** [community/community#62651] [Actions]
### Select Topic Area

Bug

### Body

My commits are visible in the repository but not in the overview. Can you solve this problem? Thank you in advance and I wish you a good work.

**Comment by Livia-Rasp** [comment]
The overview only shows commits on the main branches of repositories. Could it be, that your commits were on other branches than that?

**Reply by eryarsoyyunusemre** [reply]
No, my commits are directly in the repo. I am using WebStorm. After resetting WebStorm, I encountered such an issue, and I am quite surprised.

**Reply by eryarsoyyunusemre** [reply]
"If you check the commit count of my repository from the sales-and-marketing-frontend part and then look at the overview section, you will understand the error. I would really appreciate it if you could assist me."

**Comment by Spixmacaw** [comment]
In Genarly cummits are used github profile for increase

**Comment by Spixmacaw** [comment]
transaction control language that is used to permanently save the changes done in the transaction in tables/databases in SQL  
😅😅😅😅
